ci/df_info.txt

```diff
@@ -0,0 +1,8 @@
+<class 'pandas.core.frame.DataFrame'>
+RangeIndex: 5 entries, 0 to 4
+Data columns (total 3 columns):
+int_col      5 non-null int64
+text_col     5 non-null object
+float_col    5 non-null float64
+dtypes: float64(1), int64(1), object(1)
+memory usage: 200.0+ bytes
```

Comment:
Is this file needed to pass the `to_excel` doctest? I think we want to think better where to place those files, or what to do in these cases. But in any case, let's leave that for a separate PR. We try to keep the PRs focused on one fix, and not mix different things. This is useful for example if we have to revert some changes because something is wrong. In that case, we don't want to revert two independent changes, because one is likely to be ok.

Other than this, the PR looks good to me. Thanks!